src/dft/backends/mklgpu/backward.cpp

```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
-* Copyright 2022 Intel Corporation
+* Copyright 2022 Codeplay Software Ltd.
```

Comment:
Have we discussed how we need to handle the Copyright for this project? What's the rule used for now?

Reply:
I've not recieved any guidance on this, so I assume copyright Codeplay.

Reply:
So is this a file we created?

Reply:
Yes.